corehq/apps/app_manager/static/app_manager/js/manage_releases.js

```diff
@@ -0,0 +1,104 @@
+            self.deactivatedOn = ko.observable(details.deactivated_on);
+            self.errorMessage = ko.observable();
+            self.dom_id = "restriction_" + self.id;
+            self.ajaxInProgress = ko.observable(false);
+            self.actionText = ko.computed(function () {
+                return (self.active() ? "Remove" : "Add");
+            });
+            self.rowBgColor = ko.computed(function () {
+                return (self.active() ? "lightblue" : "lightgrey");
+            });
+            self.toggleStatus = function () {
+                self.active(!self.active());
+            };
+            self.error = ko.observable();
+            self.requestUrl = function () {
+                if (self.active()) {
+                    return initialPageData.reverse('deactivate_release_restriction', self.id);
+                }
+                return initialPageData.reverse('activate_release_restriction', self.id);
+            };
+            self.toggleRestriction = function () {
+                self.ajaxInProgress(true);
+                var oldStatus = self.active();
+                $.ajax({
+                    method: 'POST',
+                    url: self.requestUrl(),
+                    success: function (data) {
+                        if (data.success) {
+                            self.toggleStatus();
+                            self.activatedOn(data.activated_on);
+                            self.deactivatedOn(data.deactivated_on);
+                            self.error(false);
+                        } else {
+                            self.active(oldStatus);
+                            self.errorMessage(data.message);
+                        }
+                    },
+                    error: function () {
+                        self.active(oldStatus);
+                    },
+                    complete: function () {
+                        self.ajaxInProgress(false);
+                        if (self.active() === oldStatus) {
+                            self.error(true);
+                        }
+                    },
+                });
+            };
+            return self;
+        };
+
+        function manageReleasesViewModel(enabledAppReleases) {
+            var self = {};
+            self.enabledAppReleases = ko.observableArray(enabledAppReleases);
+            return self;
+        }
+        var enabledAppReleases = _.map(initialPageData.get('enabled_app_releases'), function (appRelease) {
+            return enabledAppRelease(appRelease);
+        });
+        var viewModel = manageReleasesViewModel(enabledAppReleases);
+        if (enabledAppReleases.length) {
+            $('#managed_releases').koApplyBindings(viewModel);
+        }
+        function manageReleaseSearchViewModel() {
+            var self = {};
+            self.search = function () {
+                var appId = $("#app_id_search_select").val();
```

Comment:
Since this is a knockout model, you could probably make these observables and have them prepopulate with the url search string.

Reply:
sorry this is not clear. 
Here, they are trying to fetch the value selected by the user to pass it into the url later.

Reply:
I don't really know what this page looks like, but if the user has done a search, does it make sense to have the search field populated with the search they just carried out? Can you add a screenshot to the PR description?

Reply:
just added a short video.
Those fields are populated back after search via the form